changelog/_template.rst

```diff
@@ -0,0 +1,38 @@
+{% for section in sections %}
+{% set underline = "-" %}
+{% if section %}
+{{section}}
+{{ underline * section|length }}{% set underline = "~" %}
+
+{% endif %}
+{% if sections[section] %}
+{% for category, val in definitions.items() if category in sections[section] and category != 'trivial' %}
```

Comment:
Just to make sure, is this an ordered dict sorted by the order the categories appear in the `pyproject.toml` file?